docs/pupernetes_reset.md

```diff
@@ -31,9 +31,10 @@ pupernetes reset default $(kubectl get ns -o name) --apply
 ### Options
 
 ```
-      --api-address string   Address for the pupernetes API ip:port (default "127.0.0.1:8989")
-  -a, --apply                Apply manifests-api after reset, useful when resetting kube-system namespace
-  -h, --help                 help for reset
+      --api-address string        Address for the pupernetes API ip:port (default "127.0.0.1:8989")
+  -a, --apply                     Apply manifests-api after reset, useful when resetting kube-system namespace
+      --client-timeout duration   timeout for reset (default 1m0s)
```

Comment:
`--client-timeout duration Maximum time waited for a pupernetes command to be executed`